project/plugins.sbt

```diff
@@ -0,0 +1,6 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.5")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "1.1")
+addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.16")
+libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.8.3"
```

Comment:
I'm a little fuzzy on how the dependency resolution works here, but would we not want `%%` for the other plugins as well? (Or doesn't the Scala version matter there?)

Reply:
That's a good question - AFAIK there is an option of passing specific `scalaVersion` to the `addSbtPlugin` but we usually do not use it, thus allowing for SBT to choose a version of Scala that SBT itself was built across, rather then the one we are building for.